packages/components/src/components/icons/Configuration.stories.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import { Meta, Story } from '@storybook/react'
+import { Configuration } from './Configuration'
+
+export default {
+  title: 'Components/icons/Configuration',
+  component: Configuration
+} as Meta
```

Comment:
Do we need to explicitly cast these as `Meta`? What happens if you drop the `as Meta` cast?

Reply:
I was following the practice that @soumita-dsi followed on her previous stories, but I found that if I drop 'as Meta' , it works fine, but it is recommended in storybook documentation to cast these "as Meta" .

Reply:
Odd that they're encouraging such a way of typing the objects. Could you link the documentation where it's stated here just to have it somewhere documented if someone asks later on? In my opinion, it's such a small thing anyways that I'm happy either way. I was mainly just wondering :)

Reply:
Here @rikukissa https://storybook.js.org/docs/react/api/csf#named-story-exports , you can see the examples.
But if you want I'll delete Casting as Meta anyway.